massa-node/base_config/config.toml

```diff
@@ -93,9 +93,9 @@
     max_known_endorsements_size = 1024
     # Maximum number of batches in the memory buffer.
     # Dismiss the new batches if overflow
-    operation_batch_buffer_capacity = 10024 
+    operation_batch_buffer_capacity = 128 
```

Comment:
reduced the buffer, since each item in the buffer can contain `10024` operations, see `network.max_operations_per_message`.

Reply:
Removed this, to be potentially dealt with separately, see https://github.com/massalabs/massa/issues/2574.